core/domain/exp_jobs_one_off_test.py

```diff
@@ -622,10 +628,13 @@ def test_migration_job_does_not_have_validation_fail_on_default_exp(self):
         self.process_and_flush_pending_tasks()
 
         # Verify the new exploration has been migrated by the job.
-        updated_exp = exp_services.get_exploration_by_id(self.NEW_EXP_ID)
-        self.assertEqual(
-            updated_exp.states_schema_version,
-            feconf.CURRENT_STATES_SCHEMA_VERSION)
+        with self.swap(
+            html_validation_service, 'get_filename_with_dimensions',
+            mock_get_filename_with_dimensions):
+            updated_exp = exp_services.get_exploration_by_id(self.NEW_EXP_ID)
+            self.assertEqual(
```

Comment:
There aren't any images in this exploration. Why does this need to be changed?

Reply:
My bad. Removed. Done